python/ray/tests/test_worker_capping.py

```diff
@@ -177,8 +177,8 @@ def f(i, start):
 
     # Assert that last_wwait / second_last ~= 2, with a healthy buffer since ci
     # is noisy.
-    assert 1 < last_wait / second_last < 4
-    assert 2 < last_wait < 4
+    assert second_last < last_wait < 4 * second_last
+    assert 7 < last_wait
```

Comment:
Removed upper bound because wait times can be longer than `10s` (if overhead is accounted as well). Happened in 1 out of 10 executions of this test locally.